### DISCUSSION TITLE
What is a voice IEM

### DISCUSSION THREAD

**Discussion by rightfullyhellish** [florisboard/florisboard#2393] [Q&A] [answered]
Keep trying to use voice to text, but it keeps telling me that i don't have a "Voice IEM"  installed.
Cant figure out What that is or how or where to find one.
Mostly because whenever I try to lookup "Voice IEM"
The only thing that pops up are earbuds
Any help would be appreciated. 

Thanks

**Comment by lm41** [comment]
can you please provide some more information about the system you are using, i.e. which android version, which app version, maybe also which hardware you are using?

**Reply by rightfullyhellish** [reply]
Android 13
App 0.3.16

Here screen shot

![Screenshot_20240204-191038](https://github.com/florisboard/florisboard/assets/158978341/e0e3e356-a283-4c96-afad-271db4ad5bc7)

**Reply by lm41** [reply]
You need to have a speech-to-text engine installed on your device.

**Reply by rightfullyhellish** [reply]
Yeah  i know 
But where do i get one 

I've tried every search i can think of

**Comment by tsiflimagas** [comment, marked as answer]
You can get information [here](https://github.com/florisboard/florisboard/pull/1587) and in the linked issue. There's also FUTO voice input app, apart from the IMEs mentioned, which I've heard works very well.
